app/main.js

```diff
@@ -60,11 +60,15 @@ function displaysY (height = 600) {
 }
 
 function createTrayIcon () {
```

Comment:
I think only macOS should be monochrome. On Linux and Windows we should have full-color stretchly logo